### PR TITLE
Update CODEOWNERS +PHP compatibility

### PR DESCRIPTION
**Pull request by Benoit382**

### CODE REVIEW

**Review by titoko — APPROVED** [review]

**Review by Oliboy50 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -7,15 +7,17 @@
             "email" : "olivier.mansour@m6.fr"
         }
     ],
+    "config": {
+        "bin-dir": "bin/"
+    },
     "require" : {
-        "php" : "^7.0"
+        "php" : "^7.4|^8.0",
+        "symfony/symfony": "^2.7|^3.0|^4.0"
```
Comment (Oliboy50) [inline composer.json:15]:
```suggestion
        "symfony/symfony": "^4.4"
```

(the others are not supported anymore)

File: composer.json [hunk 2]
```diff
@@ -7,15 +7,17 @@
...
     },
     "require-dev" : {
-        "atoum/atoum" : "^2.8|^3.0",
+        "atoum/atoum" : "^3.0|^4.0",
```
Comment (Oliboy50) [inline composer.json:18]:
```suggestion
        "atoum/atoum" : "^4.0",
```

no need to support an older version of atoum for php 7.4+

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ composer install --dev
 
 And you can launch coke
 ```bash
-./vendor/bin/coke
+bin/coke
```
Comment (Oliboy50) [inline CONTRIBUTING.md:21]: coke is not present anymore, we can talk about php-cs-fixer instead

**Review by Oliboy50 — APPROVED** [review]

**Comment by Benoit382** [comment]
to be archived

**Pull request closed without merging**